src/content/en/updates/2017/04/headless-chrome.md

```diff
@@ -335,13 +343,40 @@ Demos
 
 ## FAQ
```

Comment:
What would you suggest for content that's relevant but doesn't merit it's own section?

I'm seeing a lot of use of FAQs:
- https://developers.google.com/web/updates/2015/08/using-requestidlecallback#faq
- https://developers.google.com/web/updates/2016/07/payment-request#faq
- https://developers.google.com/web/updates/2016/04/scroll-anchoring#faq
- https://developers.google.com/web/updates/2015/08/using-web-app-manifest-to-set-solid-color-loading-screen#faq
- https://developers.google.com/web/updates/2015/03/increasing-engagement-with-app-install-banners-in-chrome-for-android#wzxhzdk12frequently_asked_questionswzxhzdk13
- https://developers.google.com/web/updates/2015/03/increasing-engagement-with-app-install-banners-in-chrome-for-android#wzxhzdk12frequently_asked_questionswzxhzdk13

What would you suggest for content that's relevant but doesn't merit it's own section?
We use FAQs all over the place:
- https://developers.google.com/web/updates/2015/08/using-requestidlecallback#faq
- https://developers.google.com/web/updates/2016/07/payment-request#faq
- https://developers.google.com/web/updates/2016/04/scroll-anchoring#faq
- https://developers.google.com/web/updates/2015/08/using-web-app-manifest-to-set-solid-color-loading-screen#faq
- https://developers.google.com/web/updates/2015/03/increasing-engagement-with-app-install-banners-in-chrome-for-android#wzxhzdk12frequently_asked_questionswzxhzdk13
- https://developers.google.com/web/updates/2015/03/increasing-engagement-with-app-install-banners-in-chrome-for-android#wzxhzdk12frequently_asked_questionswzxhzdk13